File: 1-beginner/the_biggest.c
#include <stdio.h>

int main()
{

    int a, b, c, the_biggest;

    scanf("%d%d%d", &a, &b, &c);

    the_biggest = (a + b + abs(a - b)) / 2;

    if (the_biggest >= c)
    {
        printf("%d eh o maior\n", the_biggest);
    }
    else
    {
        printf("%d eh o maior\n", c);
    }

    return 0;
}

File: 1-beginner/the_biggest.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)

the_biggest = int((a + b + abs(a - b)) / 2)

if the_biggest >= c:
    print(f"{the_biggest} eh o maior")

else:
    print(f"{c} eh o maior")